test/txsim/run.go

```diff
@@ -18,6 +18,11 @@ import (
 
 const DefaultSeed = 900183116
 
+const (
+	grpcMaxRecvMsgSize = 128 * 1024 * 1024
+	grpcMaxSendMsgSize = 128 * 1024 * 1024
```

Comment:
it's not immediately obvious what these values equal

```suggestion
	mebibyte = 1_048_576 // bytes
	grpcMaxRecvMsgSize = 128 * mebibyte
	grpcMaxSendMsgSize = 128 * mebibyte
```